### ISSUE TITLE
hash_indexing_failures is missing from FT.INFO

### ISSUE THREAD

**Issue by abrookins** [RediSearch/redisearch-go#106]
hash_indexing_failures is missing from the FT.INFO call https://github.com/RediSearch/redisearch-go/blob/f79df23649c64380ee74b4b8912c47a1049ef0f2/redisearch/index.go#L8 

We should include the field IMO. From a user report in Discord.

**Comment by mikeblum** [comment]
:wave: here is the code I'm using as a workaround targeting `FT.INFO` directly:

```
func (s *Client) awaitIndex() error {
	// Wait for all documents to be indexed
	var info *redisearch.IndexInfo
	var err error
	var hashIndexingFailures int
	info, err = s.idx.Info()
	for info.IsIndexing {
		time.Sleep(time.Second)
		info, err = s.idx.Info()
	}
	// hash_indexing_failures missing from *redisearch.IndexInfo
	rds := config.NewRedisClient()
	res, _ := rds.Do(context.TODO(), "FT.INFO", idxShodanRedisVersionByCityCountryGeo).Result()
	props := res.([]interface{})
	for i, itr := range props {
		prop, ok := itr.(string)
		if ok && strings.EqualFold(prop, attrHashIndexingFailures) {
			hashIndexingFailures, err = strconv.Atoi(props[i+1].(string))
			break
		}
	}
	if hashIndexingFailures > 0 {
		err = fmt.Errorf("failed to index %d documents - check for mssing indexed attributes", hashIndexingFailures)
	}
	return err
}

**Comment by filipecosta90** [comment]
@mikeblum @abrookins the PR #107 should fix it. Included positive and negative test for confirming it.

**Comment by filipecosta90** [comment]
@mikeblum @abrookins this can now be used as of version [`v1.1.1`](https://github.com/RediSearch/redisearch-go/releases/tag/v1.1.1). Thank you for raising this.

**Comment by mikeblum** [comment]
@filipecosta90 Thank you for the very prompt fix! I'll pull down `1.1.1` and rip out my hacky workaround. 🥳
